app/src/main/res/values-w600dp/scale_bar_config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="scaleBarRatio" format="float" type="fraction">0.2</item>
+    <dimen name="scaleBarMarginRight">60dp</dimen>
+    <dimen name="scaleBarMarginBottom">8dp</dimen>
+    <!-- "bottom | right" See plugin-scalebar/src/main/res/values/attrs.xml -->
+    <integer name="scaleBarGravity">0x55</integer>
```

Comment:
are we actually suggesting the users to have similar config? looks not very user-friendly tbh 😄

Reply:
I am not aware of any other way to use resources to define gravity.